reflect_abi_patch.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"maps"
+	"os"
+	"slices"
+	"strings"
+)
+
+func abiNamePatch(path string) (string, error) {
+	data, err := os.ReadFile(path)
+	if err != nil {
+		return "", err
+	}
+
+	find := `return unsafe.String(n.DataChecked(1+i, "non-empty string"), l)`
+	replace := `name := unsafe.String(n.DataChecked(1+i, "non-empty string"), l)
+	return _realName(name)`
```

Comment:
make these two strings multi-line just like you did for realname - for the `find` you can use `strings.TrimSpace` for it to still work for `strings.Replace`

Reply:
also, why not replace with a single line too, which is less prone to causing conflicts with other `name` declarations?

```
return _realName(unsafe.String(n.DataChecked(1+i, "non-empty string"), l))
```

Reply:
What exactly do you mean by make this multiline?